### PR TITLE
fvwm3: adding manpages

### PR DESCRIPTION
**Pull request by tgaudin**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by abenson** [comment]
Please follow [CONTRIBUTING.md](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md) for the commit message.  `revision` also needs to be bumped to 2.

Don't close this PR, just `git commit --amend` edit the message, save/quit, and then `git push -f`.

File: srcpkgs/fvwm3/template [hunk 1]
```diff
@@ -4,7 +4,8 @@ version=1.0.4
 revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config automake"
-makedepends="libXrandr-devel libbson-devel libevent-devel libXt-devel
+configure_args=--enable-mandoc
```
Comment (abenson) [inline srcpkgs/fvwm3/template:7]: Quote this just to be consistent.

File: srcpkgs/fvwm3/template [hunk 2]
```diff
@@ -1,10 +1,11 @@
 # Template file for 'fvwm3'
 pkgname=fvwm3
 version=1.0.4
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="pkg-config automake"
-makedepends="libXrandr-devel libbson-devel libevent-devel libXt-devel
+configure_args="--enable-mandoc"
```
Comment (abenson) [inline srcpkgs/fvwm3/template:7]: Whoops, I missed this, move it above `hostmakedepends` then it should be good!

**Comment by paper42** [comment]
could you add the email you used for the commit to your github profile?

**Comment by tgaudin** [comment]
Yes, that's done!

**Comment by ericonr** [comment]
This should have been in hostmakedeps, actually :/

**Comment by tgaudin** [comment]
> This should have been in hostmakedeps, actually :/

Not sure I understand that. Should I move asciidoc to hostmakedeps? If yes, do you want me to create a new pull request, bump the pkg version and so on?

**Comment by ericonr** [comment]
I fixed it in https://github.com/void-linux/void-packages/commit/86f9bbe105e5ced5ccbbed1543110b9d5b8f9c83

It doesn't require a pkg bump

**Comment by ericonr** [comment]
But thanks for offering c:

**Pull request merged**